scripts/generate/run.js

```diff
@@ -39,9 +39,11 @@ for (const [file, template] of Object.entries({
   'utils/structs/EnumerableMap.sol': './templates/EnumerableMap.js',
   'utils/SlotDerivation.sol': './templates/SlotDerivation.js',
   'utils/StorageSlot.sol': './templates/StorageSlot.js',
+  'utils/StorageSlotTransient.sol': './templates/StorageSlotTransient.js',
```

Comment:
I'm think "Transient" and "Storage" should go together. Also this is not a "transeient-variant" of the StorageSlot library (because it uses different type, different functions, ...)

So I'd be in vafor of changing the name. Proposal include:
- [ ] `Transient.sol`
- [ ] `TransientSlot.sol`
- [ ] `TransientStorageSlot.sol`
- [ ] Something else ?